packages/quick_actions/quick_actions/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.0+3
+
+* make const constructor for QuickActions.
```

Comment:
Maybe this can be a bit more explanatory:
```suggestion
* Added a `const` constructor for the `QuickActions` class, so the plugin will behave as documented in the  sample code mentioned in the [README.md](https://github.com/flutter/plugins/blob/59e16a556e273c2d69189b2dcdfa92d101ea6408/packages/quick_actions/quick_actions/README.md).
```